backend/src/controllers/EventController.ts

```diff
@@ -95,6 +96,10 @@ export class EventController {
   @Get('/:eventID')
   @ResponseSchema(EventResponse)
   async getEvent(@Param('eventID') eventID: number): Promise<EventResponse> {
+    logger.info({
+      message: 'getting event ' + eventID,
+      customTags: 'event',
```

Comment:
could we take this out for now before we come up with a standardized way of approaching tags?

Reply:
done.